deps/chakrashim/src/v8string.cc

```diff
@@ -101,12 +101,19 @@ int String::Write(uint16_t *buffer, int start, int length, int options) const {
 int String::WriteOneByte(
     uint8_t* buffer, int start, int length, int options) const {
   size_t count = 0;
-  if (JsCopyString((JsValueRef)this, start, length,
-                    (char*)buffer, &count) == JsNoError) {
+  // JSRT does not have a start parameter on JsCopyString, use
```

Comment:
To clarify, v8 has 3 APIs, utf8 version is special and has no start parameter, utf16 and onebyte versions have. JSRT currently only supports 2 APIs. Current JsCopyString is utf8 version and has no start parameter, same as previous JsCopyStringUtf8.